resources/js/components/user/Edit/EditForm.vue

```diff
@@ -0,0 +1,162 @@
+                        >
+                            <v-text-field
+                                id="password"
+                                label="New Password *"
+                                name="password"
+                                prepend-icon="mdi-lock"
+                                type="password"
+                                v-model="user.password"
+                                :error-messages="errors"
+                                v-on:keyup.enter="onClickEditButton"
+                                outlined
+                                required
+                                autocomplete="off"
+                            />
+                        </ValidationProvider>
+                    </v-form>
+                </v-card-text>
+                <v-card-actions>
+                    <v-spacer />
+                    <v-btn
+                        color="primary"
+                        :disabled="isEditBtnDisabled"
+                        @click="onClickEditButton()"
+                        :loading="isBtnLoading"
+                    >
+                        Edit
+                    </v-btn>
+                </v-card-actions>
+            </v-card>
+        </v-col>
+    </v-row>
+</template>
+
+<script>
+import {
+    ValidationProvider,
+    ValidationObserver
+} from "vee-validate/dist/vee-validate.full";
+export default {
+    name: "EditForm",
+    components: {
+        ValidationProvider,
+        ValidationObserver
+    },
+    data() {
+        return {
+            userEmail: "",
+            user: {
+                name: "",
+                email: "",
+                password: ""
+            },
+            users: [],
+            isBtnLoading: false,
+            errorEmail: ""
+        };
+    },
+    created() {
+        axios.get("/api/users/" + this.$route.params.id).then(res => {
+            this.user = res.data;
+            this.userEmail = res.data.email;
+        });
+        axios.get("/api/users").then(res => {
+            this.users = res.data;
+        });
+    },
+    computed: {
+        isEditBtnDisabled() {
+            if (!this.user.name || !this.user.password || !this.user.email) {
+                return true;
+            } else {
+                return false;
+            }
+        }
+    },
+    methods: {
+        async onClickEditButton() {
+            if (this.isEditBtnDisabled === true) {
+                return false;
+            }
+
+            this.isBtnLoading = true;
+
+            if (this.user.email === this.userEmail) {
+            } else {
+                for (let i = 0; i < this.users.length; i++) {
+                    if (this.user.email === this.users[i].email) {
+                        window.alert(
+                            "The email has already been used. Please use another email."
+                        );
+                        this.isBtnLoading = false;
+                        return;
+                    }
```

Comment:
Same the above comment about validation.
Please refer: https://laravel.com/docs/8.x/validation#rule-unique